drivers/1wire/ds28e17.c

```diff
@@ -537,11 +537,12 @@ static int ds_i2c_setfrequency(FAR struct ds_i2c_inst_s *inst,
         break;
 
       default:
-        i2cerr("ERROR: bad I2C freq %lu\n", frequency);
+        i2cerr("ERROR: bad I2C freq %" PRIx32 "\n", frequency);
```

Comment:
Note: This changes the output from base-10 to base-16 (which is okay as long as it's intentional).

Reply:
changed back to base-10